modules/local_storage/src/LocalStockServiceConfig.php

```diff
@@ -34,17 +35,17 @@ public function __construct(EntityTypeManagerInterface $entity_type_manager) {
   /**
    * {@inheritdoc}
    *
-   * @ToDo Do we really need a primary transaction location. If so users need a way to set this.
    */
-  public function getPrimaryTransactionLocation(PurchasableEntityInterface $entity, $quantity) {
-    $locations = $this->getLocationList($entity);
+  public function getTransactionLocation(Context $context, PurchasableEntityInterface $entity, $quantity) {
+    // location.
+    $locations = $this->getAvailabilityLocations($context, $entity);
     return empty($locations) ? NULL : array_shift($locations);
   }
 
   /**
    * {@inheritdoc}
    */
-  public function getLocationList(PurchasableEntityInterface $entity) {
+  public function getAvailabilityLocations(Context $context, PurchasableEntityInterface $entity) {
```

Comment:
The current implementation of the method is a temporary one and is basically saying that "All active locations have available stock" this commit is not changing the behavior only cleaning the interfaces and the methods implementing them. We will then move on to create configuration options that will allow setting the available locations by store.